x/tokenomics/types/message_update_param.go

```diff
@@ -47,6 +47,9 @@ func (msg *MsgUpdateParam) ValidateBasic() error {
 	switch msg.Name {
 	case ParamComputeUnitsToTokensMultiplier:
 		return msg.paramTypeIsInt64()
+	case ParamChainComputeUnitsToTokensMultiplier:
+		// see note in tx.proto
```

Comment:
1. Make sure to have TODO_IN_THIS_PR if this comment is temporary
2. #PUC (Please Update Comment) of what we want to see in tx.proto
3. Consider providing full relative path to tx.proto to simplify it for future devs